common/test/recorder/HttpRecorder.scala

```diff
@@ -16,13 +16,16 @@ import scala.concurrent.Future
 import scala.io.Source
 
 
-trait
-HttpRecorder[A] extends ExecutionContexts {
+trait HttpRecorder[A] extends ExecutionContexts {
 
   def baseDir: File
 
+  final def load(url: String, headers: Map[String, String] = Map.empty)(fetch: => Future[A]): Future[A] =
+    loadFile(url, headers)(fetch).map(toResponse _ compose contentFromFile _)
```

Comment:
I am not sure what `toResponse _ compose contentFromFile _` bring over `file => toResponse(contentFromFile(file))`
I found the latter easier to read :)